en/docs/guides/tenants/customize-automated-mails.md

```diff
@@ -0,0 +1,187 @@
+            `{{user.claim.homeAddress}}`.
+    
+        Before sending the email, WSO2 Identity Server will retrieve the user details from the user store configured for this tenant and replace the placeholders with the corresponding values of the user.
+        
+        !!! note
+            When you are using an identity claim as a placeholder in the **Account Confirmation** template
+            for User Self Registration, you need to use the notation `{{claimName}}` instead of using 
+            `{{user.claim.identity.claimName}}`. Additionally you need to send the respective value 
+            for the placeholder in the API request payload, under the `properties` section. 
+            A sample request payload is given below.
+            ```
+            {
+            ...
+            "properties": [
+            ...
+                {
+                "key": "claimName",
+                "value": "Value for the claim"
+                }
+            ...
+            ]
+            }
+            ```
+    
+
+5.  Click **Add** to add a new email template.
+
+## Deleting an email template type
+
+!!! warning
+    
+    This deletes all the email templates that correspond to the selected email template type together with the email template type.
+    
+
+1.  Sign in to WSO2 Identity Server [Management Console](../../../deploy/get-started/get-started-with-the-management-console/).
+2.  On the **Main** tab, click **Manage > Email Templates > List**.
+3.  Select the email template type.
+4.  Click **Delete Template Type** and click **ok.**
+
+## Editing an email template
+
+1.  Sign in to WSO2 Identity Server [Management Console](../../../deploy/get-started/get-started-with-the-management-console/).
+2.  On the **Main** tab, and click **Manage > Email Templates > List**.
+3.  Select/enter the email template details based on the fields that you
+    want to update in the email template.  
+
+    -   **Select Email Template Type** : You can select the type of email template you wish to customize using this dropdown.  
+        ![select-email-template-type](../../../assets/img/guides/select-email-template-type.png)
+    -   **Subject** : This is the subject of the email that is sent to the user.
+    -   **Email Body** : This is the body of the email that is sent to the user. Custom information like `{first-name}` and `{user-id}` are populated from the user store that is configured for this tenant.
+    -   **Email Footer** : This is the footer of the email.
+
+        !!! note
+            Ensure that you do not use the "\|" character in your
+            **Subject**, **Email Body** or **Email Footer** as this is not
+            supported.
+        
+
+    ![edit-email-template](../../../assets/img/guides/edit-email-template.png) 
+
+    !!! tip "Placeholders and Custom Claim Dialects"
```

Comment:
is this the same tip as given in `Add an email template` section? If so shall we use a fragment?